fastchat/model/model_adapter.py

```diff
@@ -49,6 +49,13 @@
 )
 
 
+ANTHROPIC_MODEL_LIST = (
+    "claude-1",
+    "claude-2",
```

Comment:
I think it's indeed `claude-v1` and `claude-2`. I know it's confusing... could you test and confirm this?

Reply:
Hi, my current resource could only test the AWS agile's Anthropic service, and `-v1` and `-v2` model ids works for me, but not `-1` and `-2`.  But I think the `claude-2` should works as it is the official name.

Reply:
Fastchat has been using `"claude-v1", "claude-instant-v1"` in `llm_judge.common.py`, and `"claude-2", "claude-instant-1"` in `model/model_adapter.py`. Seems that there have been different users using different sources services.

Would you consider accept all these names and let users to decide which key they choose to pass?

For reference [the official Anthropic's doc](https://docs.anthropic.com/claude/reference/selecting-a-model) shows
Family | Latest major version | Latest full version
-- | -- | --
Claude Instant | claude-instant-1 | claude-instant-1.2
Claude | claude-2 | claude-2.0

Reply:
I just confirmed that `claude-1` works. thanks. then we're good now. I think `v1` has become legacy names now.